### PR TITLE
Add plant dna manipulator board to wild mutation experiment reward

### PR DESCRIPTION
**Pull request by yorii**
## About The Pull Request

Adds the Plant DNA Manipulator circuit board to the tech node rewarded from the wild mutation experiment.

## Why It's Good For The Game

Since the plant manipulator is only available on ~~Metastation~~ some station, I think it should at least be added as a science reward to be constructed later in the round on the other maps too. 
Plus allows you to reconstruct it in case it gets destroyed on whatever station already has it.

## Changelog

:cl:
add: Added Plant DNA Manipulator circuit board to techweb node.
/:cl:

### CODE REVIEW

**Review by BurgerLUA — APPROVED** [review]

**Comment by StrangeWeirdKitten** [comment]
This PR is redundant, specifically with ``modular_zubbers\code\modules\research\techweb\all_nodes.dm`` already having the techweb things necessary, though uncommented out. ``C:\Users\gotnu\Documents\GitHub\Bubberstation\modular_zubbers\code\modules\research\designs\misc_designs.dm`` is also unticked for whatever reason. 

All that has to happen for everything to get into the techweb is for the code to be enabled. Uncomment the techweb node I pointed out and tick ```misc_designs.dm``` so it's included in the compile.

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1482,6 +1482,7 @@
 		"portaseeder",
 		"seed_extractor",
 		"adv_watering_can",
+		"plantgene",
```
Comment (StrangeWeirdKitten) [inline code/modules/research/techweb/all_nodes.dm:1485]: Go look at ``modular_zubbers\code\modules\research\techweb\all_nodes.dm`` and remove this.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (StrangeWeirdKitten) [reply code/modules/research/techweb/all_nodes.dm:1485]: Also in the future make sure you document Bubberstation edits with a comment

**Comment by Majkl-J** [comment]
Stale. Ask for a reopen when work on this begins again

**Comment by yorii** [comment]
> This PR is redundant, specifically with `modular_zubbers\code\modules\research\techweb\all_nodes.dm` already having the techweb things necessary, though commented out. `modular_zubbers\code\modules\research\designs\misc_designs.dm` is also unticked for whatever reason.
> 
> All that has to happen for everything to get into the techweb is for the code to be enabled. Uncomment the techweb node I pointed out and tick `misc_designs.dm` so it's included in the compile.

Thank you very much! And I'm sorry for forgetting this PR and letting it go stale, I'll get it up to par right away :)

**Comment by yorii** [comment]
Please re-open when anyone sees this

**Review by Majkl-J — APPROVED** [review]
Looks fine

**Pull request merged**